worker-workflow/src/test/resources/input-document-no-subdoc-with-processing-worker-versions-field-3.json

```diff
@@ -0,0 +1,30 @@
+{
+    "document": {
+        "reference": "ref_1_file",
+        "fields": {
+            "CAF_WORKFLOW_ACTION": [{
+                    "data": "family_hashing"
+                }
+            ],
+            "CAF_WORKFLOW_NAME": [{
+                    "data": "example_workflow",
+                    "encoding": "utf8"
+                }
+            ],
+            "FAILURES": [],
+            "PROCESSING_WORKER_VERSIONS": [
+                {
+                    "data": "[{\"NAME\":\"worker-base\", \"VERSION\":\"3.3.0-SNAPSHOT\"}, {\"NAME\":\"worker-entityextract\", \"VERSION\":\"4.1.0-SNAPSHOT\"}]"
```

Comment:
You are creating a single-value field where the field value is a JSON array of objects, instead of a multi-value field where each field value is a JSON object.